src/Lykke.Service.Qtum.Api/Modules/ServiceModule.cs

```diff
@@ -16,6 +20,19 @@ public ServiceModule(IReloadingManager<AppSettings> appSettings)
         protected override void Load(ContainerBuilder builder)
         {
             // Do not register entire settings in container, pass necessary settings to services which requires them
+
+            // Network setup
+            QtumNetworks.Register();
+            builder.RegisterInstance(Network.GetNetwork(_appSettings.Nested(s => s.Network).CurrentValue)).As<Network>();
+            
+            // Services setup
+            builder.RegisterType<AssetService>()
+                .As<IAssetService>();
+            
+            builder.RegisterType<BlockchainService>()
+                .As<IBlockchainService>()
+                .WithParameter("networkType", _appSettings.Nested(s => s.Network).CurrentValue);
```

Comment:
Why it doesn't use network from DI-container that was registered before?

Reply:
fixed